images/shellcheck/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
+LABEL build-date= \
+    com.redhat.build-host= \
+    description="This image provides shecllcheck binary." \
+    distribution-scope="public" \
+    io.k8s.description="This image provides shecllcheck binary." \
+    io.k8s.display-name="shellcheck" \
+    maintainer="Pipelines Service" \
+    name="shellcheck" \
+    release="0.1" \
+    summary="Provides the 0.7.1 shecllcheck" \
+    url="https://github.com/openshift-pipelines/pipelines-service/tree/main/images/shellcheck" \
+    vcs-ref=  \
+    vcs-type="git" \
+    vendor="Pipelines Service" \
+    version="0.1"
+WORKDIR /
+RUN mkdir /workspace && chmod 777 /workspace && chown 65532:65532 /workspace
+ENV HOME /tmp/home
+RUN mkdir $HOME && chmod 777 $HOME && chown 65532:65532 $HOME
```

Comment:
Can we have as few `RUN` inside dockerfile as possible. It is because -
```
The RUN instruction will execute any commands in a new layer on top of the current image and commit the results. The resulting committed image will be used for the next step in the Dockerfile.

Layering RUN instructions and generating commits conforms to the core concepts of Docker where commits are cheap and containers can be created from any point in an image’s history, much like source control.
```
You can refer this as well https://docs.docker.com/engine/reference/builder/#run